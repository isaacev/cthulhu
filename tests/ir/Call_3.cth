fn add(a: Int, b: Int) -> Int {
  a + b
}

#[entry]
fn main() -> () {
  add(1, 2, 3);
}
