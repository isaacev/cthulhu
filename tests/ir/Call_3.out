
  CALL TO NON-FUNCTION  ------------------------------------------------------

  A value of type `Int` was called as a function:

    : TEST_DIR/ir/Call_3.cth
  7 | add(1, 2, 3);
    | ^^^^^^^^

  An argument with the type `Int` was passed:

    : TEST_DIR/ir/Call_3.cth
  7 | add(1, 2, 3);
    |           ^

